lib/lib_rss.php

```diff
@@ -823,25 +823,17 @@ function errorMessageInfo($errorTitle, $error = '') {
 		$details = "<pre>{$details}</pre>";
 	}
 
+	header('HTTP 1.1 500 Internal Server Error', true, 500);
```

Comment:
The function `errorMessageInfo()` returns some text so it should best not emit headers. I suggest to move this line in the `killApp()` function.

Reply:
We could throw a `Minz_Exception` in https://github.com/FreshRSS/FreshRSS/blob/edge/p/i/index.php#L65-L69 instead of the two last lines

Reply:
Could you please support here with a change/push?

Reply:
Something like that https://github.com/FreshRSS/FreshRSS/pull/4465/commits/53ee1d1b6801e72230403ec354b11cbe859e0e2d (not tested)